omod-2.4/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/PatientResource1_8.java

```diff
@@ -195,6 +195,30 @@ public DelegatingResourceDescription getCreatableProperties() {
 		DelegatingResourceDescription description = new DelegatingResourceDescription();
 		description.addRequiredProperty("person");
 		description.addRequiredProperty("identifiers");
+
+		// we apply these generically to resources in the DelegationCrudResource before calling setConvertedProperties,
+		// but the PatientResource calls setConvertedProperties directly in getPatient method below, so we need to add this here
```

Comment:
fyi @rkorytkowski @ibacher as to why we need to apply all the audit properties here on Patient